qframelesswindow/unix/__init__.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, parent=None):
 
         self.titleBar.raise_()
         self.resize(500, 500)
+        self.setContentsMargins(0, self.titleBar.height(), 0, 0)
```

Comment:
This line should be added by the user. The title bar of some apps will not take up space, for example:
![image](https://user-images.githubusercontent.com/65096819/178928894-90066f16-d652-42ff-b1ee-cd4c69006163.png)